app/src/main/java/no/usn/mob3000/ui/components/auth/PasswordResetContent.kt

```diff
@@ -97,6 +99,7 @@ fun PasswordResetContent(
                     onValueChange = { confirmPassword = it },
                     label = { Text(stringResource(R.string.auth_reset_password_confirm)) },
                     placeholder = { Text(stringResource(R.string.auth_login_password_placeholder)) },
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[Same as this](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).